packages/ember-routing-htmlbars/tests/helpers/element_action_test.js

```diff
@@ -1002,6 +1085,29 @@ QUnit.test('should respect preventDefault=false option if provided', function()
   equal(event.isDefaultPrevented(), false, 'should not preventDefault');
 });
 
+QUnit.test('should respect preventDefault=false option if provided bound', function() {
+  view = EmberView.create({
```

Comment:
I agree with both here. :smiley_cat: 

Lets land to match the tests in the surrounding area, and then work to actually fix the test suite.